packages/@webex/plugin-meetings/src/meeting/muteState.ts

```diff
@@ -125,37 +125,38 @@ export class MuteState {
   }
 
   /**
-   * Mutes/unmutes local track
+   * Mutes/unmutes local stream
    *
    * @param {Object} meeting - the meeting object
-   * @param {Boolean} mute - true to mute the track, false to unmute it
+   * @param {Boolean} mute - true to mute the stream, false to unmute it
    * @param {ServerMuteReason} reason - reason for muting/unmuting
    * @returns {void}
    */
-  private muteLocalTrack(meeting: any, mute: boolean, reason: ServerMuteReason) {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
this doesn't seem to be needed for anything.... all vars are used in this method, aren't they? Also having this eslint comment here causes a warning about jsdocs missing.

Reply:
Done in #3053